spacy/cli/_util.py

```diff
@@ -556,3 +602,121 @@ def setup_gpu(use_gpu: int, silent=None) -> None:
         local_msg.info("Using CPU")
         if has_cupy and gpu_is_available():
             local_msg.info("To switch to GPU 0, use the option: --gpu-id 0")
```

Comment:
These methods have been moved here from `run.py` as they are now also called from `parallel.py`.

Reply:
Are there any changes in `check_rerun()`, `update_lockfile()`, `check_deps()`, `_get_lock_entry()`, `_get_fileinfo()`?

Reply:
OK, sorry, I see now that my original comment wasn't really very helpful :-). So I can confirm there are no changes to the behaviour of any of this code, it's unchanged except for:

- the names of `get_lock_entry()` and `get_fileinfo()` have been changed to `_get_lock_entry()` and `_get_fileinfo()`
- `check_deps()` has been moved into a separate function (it was previously part of `run.project.run()`
- within `check_deps()` the name of the `subcommand` variable has been changed to `cmd_name`